scripts/build.py

```diff
@@ -524,7 +614,7 @@ def save_hashes_as_json(target_filepath, file_hashes):
             path would also contain hash.
         file_hashes: dict(str, str). Dictionary of file hashes.
     """
-
+    ensure_directory_exists(HASHES_JSON)
```

Comment:
Should we rename the method since it can also check for file existence?

Reply:
`ensure_directory_exists` only makes sure that the __parent__ dir of HASHES_JSON(`/build/assets/hashes.js`) exists, which is `/build/assets`.

`ensure_files_exists` does check for file existence though.

Reply:
Ok.